src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -2615,13 +3329,17 @@ subroutine internal_tides_init(Time, G, GV, US, param_file, diag, CS)
   allocate(CS%frequency(num_freq))
 
 
+  call get_param(param_file, mdl, "DEBUG", CS%debug, &
+                 "If true, write out verbose debugging data.", &
+                 default=.false., debuggingParam=.true.)
+
   ! The periods of the tidal constituents for internal tides raytracing
   call read_param(param_file, "TIDAL_PERIODS", periods)
 
   do fr=1,num_freq
     period = extract_real(periods, " ,", fr, 0.)
     if (period == 0.) call MOM_error(FATAL, "MOM_internal_tides: invalid tidal period")
-    CS%frequency(fr) = 8.0*atan(1.0)/period
+    CS%frequency(fr) = 8.0*atan(1.0)/(US%s_to_T*period)
```

Comment:
With the units of period documented in the comments of its declaration, the multiplication by US%s_to_T should occur on line 3340, which would then read `period = US%s_to_T*extract_real(periods, " ,", fr, 0.)`, and not on line 3342.

Reply:
done